src/materialized/src/bin/coordd.rs

```diff
@@ -81,19 +81,35 @@ async fn run(args: Args) -> Result<(), anyhow::Error> {
 
     let dataflow_client = dataflowd::RemoteClient::connect(&args.dataflowd_addr).await?;
 
+    let experimental_mode = false;
     let mut metrics_registry = MetricsRegistry::new();
+    let coord_storage = coord::catalog::storage::Connection::open(
+        &args.data_directory.join("catalog"),
+        Some(experimental_mode),
+    )?;
+    let persister = coord::PersistConfig::disabled()
+        .init(
+            // TODO(benesch): if we were enabling persistence, it would NOT be
+            // ok to use `cluster_id` as the reentrance here because in theory
+            // may be multiple `dataflowd`s using the cluster ID. But at the
```

Comment:
I'm confused why this TODO talks about dataflowd when the file is bin/coordd.rs. is there something I'm missing?

Reply:
> is there something I'm missing?

Not really. I was attempting to construct some hypothetical situation whereby the `dataflowd`'s started persistence with the same cluster ID. I've rewritten the comment based on your suggestion above to create a random UUID.